@xen-orchestra/web/src/components/site/pools/PoolsSidePanel.vue

```diff
@@ -44,50 +44,32 @@
         </UiCardTitle>
         <div class="content">
           <!-- Pool -->
-          <VtsCardRowKeyValue>
-            <template #key>{{ t('pool') }}</template>
+          <UiLabelValue :label="t('pool')" :copy-value="server.poolId" ellipsis>
             <template #value>
               <UiLink
                 v-if="server.poolId !== undefined && server.poolNameLabel !== undefined"
                 icon="fa:city"
                 size="small"
                 :to="`/pool/${server.poolId}/`"
+                class="link"
               >
-                {{ server.poolNameLabel }}
+                <div v-tooltip class="text-ellipsis">
+                  {{ server.poolNameLabel }}
+                </div>
               </UiLink>
             </template>
-            <template v-if="server.poolId !== undefined" #addons>
-              <VtsCopyButton :value="server.poolId" />
-            </template>
-          </VtsCardRowKeyValue>
+          </UiLabelValue>
           <!-- ID -->
-          <VtsCardRowKeyValue>
-            <template #key>{{ t('id') }}</template>
-            <template #value>{{ server.id }}</template>
-            <template #addons>
-              <VtsCopyButton :value="server.id" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('id')" :value="server.id" :copy-value="server.id" ellipsis />
           <!-- Description -->
-          <VtsCardRowKeyValue>
-            <template #key>{{ t('description') }}</template>
-            <template #value>{{ server.poolNameDescription }}</template>
-            <template v-if="server.poolNameDescription !== undefined" #addons>
-              <VtsCopyButton :value="server.poolNameDescription" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue
+            :label="t('description')"
+            :value="server.poolNameDescription"
+            :copy-value="server.poolNameDescription"
+            ellipsis
+          />
           <!-- tag -->
-          <VtsCardRowKeyValue>
-            <template #key>{{ t('tags') }}</template>
-            <template #value>
-              <UiTagsList v-if="pool !== undefined && pool.tags.length > 0">
-                <UiTag v-for="tag in pool.tags" :key="tag" accent="info" variant="primary">{{ tag }}</UiTag>
-              </UiTagsList>
-            </template>
-            <template v-if="pool !== undefined && pool.tags.length > 0" #addons>
-              <VtsCopyButton :value="pool.tags.join(', ')" />
-            </template>
-          </VtsCardRowKeyValue>
+          <UiLabelValue :label="t('tags')" :value="pool?.tags" :copy-value="pool?.tags" ellipsis />
```

Comment:
```suggestion
          <UiLabelValue :label="t('tags')" :value="pool?.tags" :copy-value="pool?.tags" />
```
I don't believe there are any ellipses for tags in sidepanel.